themes/openy_themes/openy_carnation/src/scss/component/_breadcrumbs.scss

```
@@ -100,4 +100,20 @@ body.without-banner {
   .breadcrumbs-wrapper {
     position: initial;
   }
+  &.page-node-type-landing-page {
+    .breadcrumbs-wrapper {
+      _margin-top: -225px;
```

@retif are You sure that we this will work?
```suggestion
      margin-top: -225px;
```

---

i need to delete that line